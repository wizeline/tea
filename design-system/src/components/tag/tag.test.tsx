import { newE2EPage } from '@stencil/core/testing';

describe('wz-tag', () => {
  it('renders value', async () => {
    const page = await newE2EPage();
    await page.setContent('<wz-tag>Hello, my name is Marty McFly</wz-tag>');
    const screenshot = await page.compareScreenshot();
    expect(screenshot).toMatchScreenshot();
  });

  it('renders tag with icon', async () => {
    const page = await newE2EPage();
    await page.setContent(
      "<wz-tag icon='edit'>Hello, my name is Marty McFly</wz-tag>",
    );
    const screenshot = await page.compareScreenshot();
    expect(screenshot).toMatchScreenshot();
  });
});
