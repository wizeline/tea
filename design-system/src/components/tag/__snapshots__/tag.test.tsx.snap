// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`wz-tag renders tag with icon 1`] = `
<wz-tag
  class="hydrated"
  icon="edit"
>
  
  
  <div
    class="tag-container neutral"
  >
    <div
      class="tag-icon-container left-container"
    >
      <wz-icon
        color="var(--ink-clear)"
        icon="edit"
        xxsmall=""
      />
    </div>
    <div
      class="tag-inner-container"
    >
      
      Hello, my name is Marty McFly
      
    </div>
  </div>
</wz-tag>
`;

exports[`wz-tag renders value 1`] = `
<wz-tag
  class="hydrated"
>
  
  
  <div
    class="tag-container neutral"
  >
    
    <div
      class="tag-inner-container"
    >
      
      Hello, my name is Marty McFly
      
    </div>
  </div>
</wz-tag>
`;
